src/seedu/addressbook/AddressBook.java

```diff
@@ -455,15 +456,25 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
     private static ArrayList<String[]> getPersonsWithNameContainingAnyKeyword(Collection<String> keywords) {
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
         for (String[] person : getAllPersonsInAddressBook()) {
-            final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person).toLowerCase()));
+            Collection<String> newKeywords= changeAllStringsToLowerCase(keywords);
+            if (!Collections.disjoint(wordsInName, newKeywords)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
 
-    /**
+    private static Collection<String> changeAllStringsToLowerCase(Collection<String> keywords) {
```

Comment:
how does this affect the way data is stored or displayed?
i haven't looked at the entire code; my observation is based on this diff; it seems to me that doing your method may hamper the way data is stored or displayed.
e.g., if i want to store/display in this format: LASTNAME, Firstname when you change the case it becomes: lastname, firstname
if